content/blog/2024-07-31-tmix-07.md

```diff
@@ -0,0 +1,77 @@
++++
+title = "This Month in Xilem, July 2024"
+authors = ["Daniel McNab", "Olivier Faure"]
++++
+
+This is our monthly-ish update for the Linebender community.
+Linebender is an informal open-source organization working on various projects to advance the state of the art in GUI for [the Rust programming language](https://rust-lang.org).
+
+This month saw a lot of community contributions to Linebender projects.
+Philipp Mildenberger especially is a major driver for progress on Xilem and especially the `xilem_web` crate.
+Bruce Mitchener has been very active reviewing pull requests, which is an under-appreciated but essential task for any open-source project.
+
+## Xilem
+
+- [xilem#423][] integrates Tokio into Xilem, showing how the Xilem model can interact with async work.
+- [xilem#428][] by Philipp adds explicit support for Flex parameters.
+- [xilem#440][] and [xilem#448] by Philipp add support for async primitives in xilem_web, namely async events and a MemoizedAwait view.
+
+## Masonry
+
+The most impactful change on the Masonry side was implementing the ["Widgets in arenas" RFC](https://github.com/linebender/rfcs/blob/main/rfcs/0006-widgets-in-arenas.md) in [xilem#396][].
+Widgets are now stored in a global arena, which has little impact on the public APIs but make future changes a lot easier to implement.
+
+- [xilem#417][] exposes an unstable API for embedding Masonry and Xilem into an existing Winit application.
+- [xilem#418][], [xilem#421][] improved our unofficial support for iOS.
+- [rfcs#7][] specifies a clearer architecture for passes, including simplifications necessary for compositor use.
+  Actually integrating with the compositor is still out of scope [for reasons Raph has laid out before](https://raphlinus.github.io/ui/graphics/2020/09/13/compositor-is-evil.html), but the new pass specification gives us a starting point when we get there.
+- [xilem#474](https://github.com/linebender/xilem/pull/474) adds a calculator example, which works with Android.
```

Comment:
I think this belongs in the Xilem section. The old Calculator example was pure Masonry. The new one is Xilem.